android/app/src/main/res/layout/content_session_detail.xml

```diff
@@ -0,0 +1,203 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v4.widget.NestedScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    app:layout_behavior="@string/appbar_scrolling_view_behavior"
+    tools:showIn="@layout/activity_sessions_detail"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:id="@+id/nested_scrollview_session_detail"
+    tools:context=".activities.SessionDetailActivity">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:paddingLeft="@dimen/content_detail_margin">
```

Comment:
Yeah, but it looks good over there as the layout for that app is a bit different than out layout. 
For example that whitespace makes sense as they are displaying the speaker's image over there and because the background colors for all elements are same.
Since we are not using that empty space, so I'd suggest to remove it for now.
We can always add it later on. 🙂